selenium/che-selenium-test/src/test/resources/conf/selenium.properties

```diff
@@ -28,11 +28,11 @@ google.password=
 # Define configuration for che default test user.
 # For che single user assembly, the default values will be enough.
 # For che multiuser assembly, user must be already created on identity provider service.
-che.test_user.email=che@eclipse.org
-che.test_user.password=secret
+che.test_user.email=test@gmail.com
```

Comment:
From security reasons it's better not to place real credentials in the public config file.

Reply:
Let's stay empty properties.